### PR TITLE
Prefer true/false to yes/no in YAML files

### LINKED ISSUE #49: Stick to `true`/`false` when using Boolean values in YAML
I would really like to move away from `yes`/`no` and just stick to `true`/`false` when using Boolean values.

_Originally posted by @mcdonnnj in https://github.com/cisagov/ansible-role-assessment-tool/pull/48#discussion_r1271088340_

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request modifies the Ansible role to prefer `true`/`false` to `yes`/`no` in YAML files.

## 💭 Motivation and context ##

Resolves #49.  See also cisagov/skeleton-ansible-role#157.

## 🧪 Testing ##

All automated tests pass.  I also ran `find . -name "*.yml" -exec grep -HniF ": yes" {} \;` and `find . -name "*.yml" -exec grep -HniF ": no" {} \;` to identify any places I may have missed.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍

**Review by jasonodoom — APPROVED** [review]
So fresh and so clean.

File: README.md
Comment (mcdonnnj) [inline README.md:1]: While we're here did we want to sort the variables in the examples provided in the README?

**Review by mcdonnnj — APPROVED** [review]
Thanks for taking care of this. I have one thought for your consideration but otherwise this LGTM ✔

**Review by dv4harr10 — APPROVED** [review]

File: README.md
Comment (jsf9k) [reply README.md:1]: Please see commit c105947.

**Pull request merged**